docs/image-based-upgrade.md

```diff
@@ -324,13 +325,14 @@ Pivot
 Post-Pivot
 
 - Before OCP is started, a systemd service will run which will restore the basic platform configuration and regenerate the platform certificates using the [recert tool](https://github.com/rh-ecosystem-edge/recert).
-- Once LCA starts it will restore the saved IBU CR.
+- Once LCA starts, it will restore the saved IBU CR.
 - Restore the remaining platform configuration.
 - Wait for the platform to recover - Cluster/day2 operators and MCP are stable.
 - Apply extra manifests that were saved pre-pivot.
-- Apply any OADP restore CRs that were saved pre-pivot. Platform artifacts will be restored first including ACM artifacts if the system is managed by ACM.
+- Apply any OADP restore CRs that were saved pre-pivot. Platform artifacts will be restored first, including ACM artifacts if the system is managed by ACM.
+- If LVMS is used, automatically restores the `.spec.persistentVolumeReclaimPolicy` field of PVs to its original (pre-pivot) value.
 
-Upon completion, the condition will be updated to "Upgrade Completed".
+Upon completion, the condition will be updated to "Upgrade Completed."
```

Comment:
nit: I think the original would be correct. Both grammatically and because the condition string doesn't include a period. Mind you, the condition type also wouldn't include a space.